dohq_artifactory/admin.py

```diff
@@ -271,6 +272,11 @@ def encryptedPassword(self):
         Method for backwards compatibility, see property encrypted_password
         :return:
         """
+        # todo remove around April 2022
```

Comment:
It costs about nothing to support these fields, let's keep it and remove TODOs

Reply:
> code is strongly polluted

You haven't seen polluted code then :D 

>but we need a clean out at some moment of time

We do not. I agree with warnings, but let's remove todo. I saw warnings from 2015 in one library, and the library still support methods.

Reply:
> I saw warnings from 2015 in one library, and the library still support methods.
and that does not mean that we need to keep code dirty. I can probably find more examples where deprecation strategy is defined, even CPython itself :)

but for now let's remove todo and reiterate in some time, maybe next year